src/EditorFeatures/VisualBasicTest/UseNullPropagation/UseNullPropagationTests.vb

```diff
@@ -251,5 +251,31 @@ Class C
     End Sub
 End Class")
         End Function
+
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseNullPropagation)>
+        Public Async Function TestNullable1() As Task
```

Comment:
c# doesn't have the same issue.  This specifically looks for 'null' for C# and we can't end up with a value type there (this is why this was missed originally).